app/models/match.rb

```diff
@@ -80,6 +80,7 @@ class Match < ApplicationRecord
     where('enemy_thrower = ? OR ally_thrower = ? OR enemy_leaver = ? OR ally_leaver = ?',
           true, true, true, true)
   end
+  scope :with_potg, ->{ where(potg: true) }
```

Comment:
This is for Trends but I don't know if there should be a chart for POTGs on the trends page, should I just remove this for now?

Reply:
Hmm if the scope isn’t in use, I’d say remove it, yeah.